build.gradle

```diff
@@ -122,11 +86,9 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
             address("localhost:10006")
             adminAddress("localhost:10046")
         }
-        webPort 10007
         cordapps = [
```

Comment:
It's a shame the new `cordapp` block isn't available in 3.x.

Reply:
Yep